apps/braze/src/fields/BasicField.ts

```diff
@@ -0,0 +1,15 @@
+import { Field } from './Field';
+
+export class BasicField extends Field {
```

Comment:
nit: I think we should add a comment specifying which are the "BasicFields" (Symbol, number, decimal, date, bool, json, Text, etc) for future reference.